--- configure.ac.orig	2020-09-09 15:00:05.000000000 +0200
+++ configure.ac	2020-09-09 15:08:10.000000000 +0200
@@ -345,7 +345,7 @@
 AM_CONDITIONAL(BUILD_WIN32, test x$bwin32 = xtrue)
 AM_CONDITIONAL(BUILD_CYGWIN, test x$cygwin = xtrue)
 AM_CONDITIONAL(BUILD_MIDIPIX, test x$midipix = xtrue)
-AM_CONDITIONAL(BUILD_WITH_NO_UNDEFINED, test x$bwin32 = xtrue || test x$cygwin = xtrue || test x$midipix = xtrue)
+AM_CONDITIONAL(BUILD_WITH_NO_UNDEFINED, test 1 = 1)
 
 if test x$bwin32 = xtrue; then
   AC_HAVE_LIBRARY([ws2_32])
